src/components/Navbar/SettingsDrawer/TranslationSelectionBody.tsx

```diff
@@ -125,14 +125,16 @@ const TranslationSelectionBody = () => {
     <div>
       <div className={styles.searchInputContainer}>
         <SpinnerContainer isLoading={isLoading}>
-          <Input
-            prefix={<IconSearch />}
+          <DrawerSearchIcon />
+          <input
+            className={styles.searchInput}
+            type="text"
             id="translations-search"
             value={searchQuery}
-            onChange={setSearchQuery}
+            onChange={(event) => {
```

Comment:
This should be a separate function e.g. `const handleOnSearchQueryChange` than an inline one.